src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/helpers.ts

```diff
@@ -28,3 +30,10 @@ export const approveTrade = async ({
 
   return txHash
 }
+
+export const isPermit2Hop = (tradeQuoteStep: TradeQuoteStep | undefined) => {
```

Comment:
Can we TODO this to generalize it vs. SwapperName heuristics? 🙏🏽

Reply:
Or perhaps SwapperName *is* the right heuristics as we add Permit2 to other swappers?

Reply:
Yeah until we know what our next expansion of this is, we should wait and see